libs/journeys/ui/src/components/Button/Button.tsx

```
@@ -298,10 +298,17 @@ export function Button({
                 '&:hover': {
-                  backgroundColor:
-                    buttonVariant === ButtonVariant.text
-                      ? 'transparent'
-                      : `${buttonColor ?? 'primary'}.main`
+                  backgroundColor: (() => {
+                    switch (buttonVariant) {
+                      case ButtonVariant.text:
+                        return 'transparent'
+                      case ButtonVariant.outlined:
+                        return `${buttonColor ?? 'transparent'}`
+                      case ButtonVariant.contained:
```

Biome (1.9.4)

[error] Useless case clause.

because the default clause is present:

Unsafe fix: Remove the useless case.


(lint/complexity/noUselessSwitchCase)